doc/nrf/ug_thread_tools.rst

```diff
@@ -37,15 +37,235 @@ Thread Border Router
 Thread Border Router is a specific type of Border Router device that provides connectivity from the IEEE 802.15.4 network to adjacent networks on other physical layers (such as Wi-Fi or Ethernet).
 Border Routers provide services for devices within the IEEE 802.15.4 network, including routing services for off-network operations.
 
-Typically, a Border Router solution consists of the following parts:
+`OpenThread Border Router`_ (OTBR) released by Google is an open-source implementation of the Thread Border Router recommended to use for development purpose.
 
-* Application based on the :ref:`thread_architectures_designs_cp_ncp` design or its :ref:`thread_architectures_designs_cp_rcp` variant compatible with the IEEE 802.15.4 standard.
-  This application can be implemented, for example, on an nRF52 device.
-* Host-side application, usually implemented on a more powerful device with incorporated Linux-based operating system.
+It minimally supports the following functions:
 
-|NCS| does not provide a complete Thread Border Router solution.
-For development purposes, you can use `OpenThread Border Router`_, an open-source Border Router implementation that you can set up either on your PC using Docker or on Raspberry Pi.
-OpenThread Border Router is compatible with Nordic Semiconductor devices.
+* Bidirectional IP connectivity between Thread and Wi-Fi/Ethernet networks.
+* Bidirectional service discovery via mDNS (on Wi-Fi/Ethernet link) and SRP (on Thread network).
+* Thread-over-infrastructure that merges Thread partitions over IP-based links.
+* External Thread Commissioning (for example, a mobile phone) to authenticate and join a Thread device to a Thread network.
+
+
+Configuring OpenThread Border Router
+====================================
+
+There are two ways in which you can set up the Thread Border Router and it depends on your use case.
+
+The recommended option is to build and configure Thread Border Router on RaspberryPi 3B or newer. This option provides
+the most functionality available in OTBR such as Border Routing capabilities (e.g. needed for establishing Thread communication with
+Mobile phone on WiFi network). 
+
+However, for the development, the easiest and quickest way is to run the OTBR Docker container on any Linux-based system.
+This solution is advisable whenever you are only intested in direct communication between your Border Router and the Thread Network, for example,
+to establish IP communication between application running on Linux (e.g. **Python Controller for Matter**) and application running on Thread node.
+
+.. tabs::
+
+   .. tab:: Manual installation
+
+      To setup and configure OpenThread Border Router, please follow the `OpenThread Border Router Codelab`_.
+
+      Note that you may also use nRF52840 Dongle or nRF52840 Development Kit as RCP device, with different transport layer (e.g. UART) instead of presented USB option.
```

Comment:
Should we mention that building the Radio Co-processor using nRF 52840 DK is one of the steps described in the code lab and it can be replaced with the dongle? It can be helpful for someone unfamiliar with the RCP concept.